modules/rosa-sts-creating-a-cluster-with-customizations.adoc

```diff
@@ -138,7 +139,13 @@ I: To watch your cluster installation logs, run 'rosa logs install -c <cluster_n
 <2> If more than one matching set of account-wide roles are available in your account for a cluster version, an interactive list of options is provided.
 <3> Multiple availability zones are recommended for production workloads. The default is a single availability zone.
 <4> Enable this option if you are using your own AWS KMS key to encrypt the control plane data volumes and the PVs for your applications. Specify the ARN for the KMS key that you added the account-wide role ARN to in the preceding step.
-<5> The output includes a custom command that you can run to create a cluster with the same configuration in the future.
+<5> Enable this option only if your use case requires etcd key value encryption in addition to the control plane storage encryption that encrypts the etcd volumes by default.
+<6> The output includes a custom command that you can run to create a cluster with the same configuration in the future.
++
+[IMPORTANT]
+====
+By enabling etcd encryption for the key values in etcd state, you might incur a performance overhead of approximately 20%. Red Hat only recommends that you enable etcd encryption in addition to the default storage-level encryption if you specifically require this for your use case.
```

Comment:
I would really like this admonition to be directly under call out 5, but could not determine how to do this. Does anyone know if this is possible? Thanks!

Reply:
This has been resolved by wrapping an open block (`--`) around the example output and the call out list. Thanks @mikemckiernan for the suggestion!